### PR TITLE
Adding the tooling to set up github codespaces

### PR DESCRIPTION
**Pull request by fathom-dave**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (fathom-parth) [inline .vscode/launch.json]: are comments supported by json?

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
-# python-interview-prep
\ No newline at end of file
+# Python Interview Prep
+
+Thanks for interviewing at Fathom. To make sure you're prepared before your interviews, we'd like to have you test out the interview coding environment in advance. This process should at most take 5 minutes.
+
+For some of our interviews we'll be using Github Codespaces to give you a repo and development environment similar to what you'd use in your day-to-day job. Please work through the following guide to make sure you understand Github Codespaces, how do use the development environment and the toolchain.
```
Comment (fathom-parth) [inline README.md]:
```suggestion
For some of our interviews we'll be using Github Codespaces to give you a repo and development environment similar to what you'd use in your day-to-day job. Please work through the following guide to make sure you understand Github Codespaces, how to use the development environment and the toolchain. 
```

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+If that didn't work for you, in this repository navigate to the `Code` button, the `Codespaces` tab then click the + sign. This will open up a codespace in a new tab.
+
+![Creating a Codespace](./img/create_codespace.png)
+
+NOTE: It may take a minute or two for the container running the development environment to build.
+
+### 2. Familiarize Yourself with VSCode
+
+In the new tab, you'll have a full fledge [Visual Studio Code](https://code.visualstudio.com/), shortened to VSCode, and container to build and test your code in. Each interview will have a different VSCode and container managed by Codespaces that will be pre-installed with all of the software you'll need to complete the set of tasks. For the purposes of interviewing you will only be allowed the extensions and tools given to you in a VSCode instance.
```
Comment (fathom-parth) [inline README.md]:
```suggestion
In the new tab, you'll have a full fledged [Visual Studio Code](https://code.visualstudio.com/), shortened to VSCode, and container to build and test your code in. Each interview will have a different VSCode and container managed by Codespaces that will be pre-installed with all of the software you'll need to complete the set of tasks. For the purposes of interviewing you will only be allowed the extensions and tools given to you in a VSCode instance.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+2. Edit the file run in 1. and insert your name in the output.
+3. Run `make test`
+    - What test broke? What file did the test break in?
+4. Fix the test
+5. Format and lint your code by running `make lint fix`
+6. Commit your changes
+
+### 4. Tear Down
+
+If you got here and completed everything, looks like you're ready! To close a codespace, simply close the tab. In the repository, click `Code`, the `Codespaces` tab then click the ellipsis sign next to your codespace name then Delete. You will be required to do this at the end of each question.
```
Comment (fathom-parth) [inline README.md]: if they don't do this, does Fathom get charged?

File: README.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+In the new tab, you'll have a full fledge [Visual Studio Code](https://code.visualstudio.com/), shortened to VSCode, and container to build and test your code in. Each interview will have a different VSCode and container managed by Codespaces that will be pre-installed with all of the software you'll need to complete the set of tasks. For the purposes of interviewing you will only be allowed the extensions and tools given to you in a VSCode instance.
+
+Take a look around and use this as a playground. Be sure you understand how to:
+- Find and navigate the file structure
+- Open a terminal
+- Use git to create commits
+
+### 3. Test Scenario
+
+If you think you're ready, try this quick task and answer the following questions.
```
Comment (fathom-parth) [inline README.md]: do they need to record the answers? I assume not but should make clear!

File: README.md (hunk 4 above)
Comment (fathom-dave) [inline README.md]: Need to refresh the page before being able to close the codespace

**Pull request merged**